config_proc_macro/Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
 name = "rustfmt-config_proc_macro"
 version = "0.3.0"
+categories = ["development-tools::procedural-macro-helpers"]
 edition = "2018"
-description = "A collection of procedural macros for rustfmt"
 license = "Apache-2.0/MIT"
-categories = ["development-tools::procedural-macro-helpers"]
 repository = "https://github.com/rust-lang/rustfmt"
+description = "A collection of procedural macros for rustfmt"
```

Comment:
Again, maybe we want to hold off on applying the Cargo.toml formatting to rustfmt until a future PR.